docs/CHANGELOG.md

```diff
@@ -12,6 +12,15 @@ For more information about `AlexsLemonade/scpca-nf` versions, please see [the re
 <!-- PUT THE NEW CHANGELOG ENTRY RIGHT BELOW THIS -->
 <!-------------------------------------------------->
 
+## PLACEHOLDER FOR RELEASE DATE
+
+* The data for all samples, other than multiplexed samples, can now be downloaded as either a `SingleCellExperiment(R)` or `AnnData(Python)` object contained in a RDS or HDF5 file, respectively.
+* The sample metadata found in `single_cell_metadata.tsv` has been updated to include ontology term ids for age, sex, organism, ethnicity, diagnosis, and tissue location, when available.
```

Comment:
Is there a section for each download type we could link to here?

Reply:
This is referring to the metadata table that's included in the download, not the metadata that's in the actual object. I added a link to that section of the docs.